pkg/graph/constructor.go

```diff
@@ -94,3 +96,53 @@ func (g *Graph) AddChannel(channel messagingv1.Channel) {
 
 	v.AddEdge(to, dest, NoTransform)
```

Comment:
I have a question here: the definition for NoTransform Function is [here](https://github.com/knative/eventing/blob/63068bd53ae6beecad395fd72792820c4dbb43f2/pkg/graph/types.go#L146).
```
func NoTransform(et *eventingv1beta3.EventType, tfc TransformFunctionContext) (*eventingv1beta3.EventType, TransformFunctionContext) {
	return et, tfc
}
```


And the definition for AddEdge function is [here](https://github.com/knative/eventing/blob/63068bd53ae6beecad395fd72792820c4dbb43f2/pkg/graph/types.go#L119).

```
func (v *Vertex) AddEdge(to *Vertex, edgeRef *duckv1.Destination, transform TransformFunction) {
	edge := &Edge{from: v, to: to, transform: transform, self: edgeRef}
	v.outEdges = append(v.outEdges, edge)
	to.inEdges = append(to.inEdges, edge)

}
```

I am just a bit confused here, will NoTransform function just be a null in this case?  And for the vertex `v`, it's `transform` field will just be a null?

Reply:
`NoTransform` just does not transform the eventtype (no changes). This is different from being null